### PR TITLE
db setup update

### PR DESCRIPTION
**Pull request by sgbstats**

### CODE REVIEW

File: Your MFT Statisticians/Materials/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Resources
 
+In the materials folder
+- Stats request form
+- HRA protocol template
+- Guidance for setting up a database
+
 [This Youtube video explains how to annoy us](https://youtu.be/Hz1fyhVOjr4)
```
Comment (RLHarwood) [inline Your MFT Statisticians/Materials/README.md:8]: We may need to run this one past Cath

File: Your MFT Statisticians/Materials/database setup.Rmd [hunk 2]
```diff
@@ -20,26 +20,45 @@ library(tidyverse)
...
 
 A good database allows easy working, scalability and transparency. 
 
-This means that someone could come onto the project and easily see what is going on; it is portable between systems; new data can be easily added; reviewers can see the steps we have taken.
+This means that: 
+- Someone could come onto the project and easily see what is going on
+- It is easy to clean and fix errors
+- New data can be easily added
+- Reviewers can see the steps we have taken.
+- It is portable between systems
```
Comment (RLHarwood) [inline Your MFT Statisticians/Materials/database setup.Rmd:35]: Does this mean between different types of computer or software?

File: Your MFT Statisticians/Materials/database setup.Rmd [hunk 3]
```diff
@@ -20,26 +20,45 @@ library(tidyverse)
...
 
 ## Subject identifiers
 
 What ever you use for your subject identifier, there must be a process of going back to the source documentation. This must be kept secure, in line with your department's or R&I's policies.
 
-If you have a simple cross sectional design with one row per subject then simply putting 1, 2, 3, ... will work quite nicely. Please save a list on your computer that links the study ID to the identifiable patient data. 
+If you have a simple cross sectional design with one row per subject then putting 1, 2, 3,... will work quite nicely. Please save a list on your computer that links the study ID to the identifiable patient data. 
 
-If you have multiple rows per subject or data is collected across multiple sheets, think about how you are going to link the patients up. Using 1, 2, 3, ... risks data not matching up. Think about using another unique identifier, e.g. hospital number.
+If you have multiple rows per subject or data is collected across multiple sheets, think about how you are going to link the patients up. Using 1, 2, 3,... risks data not matching up. Think about using another unique identifier, e.g. hospital number.
```
Comment (RLHarwood) [inline Your MFT Statisticians/Materials/database setup.Rmd:55]: Are they allowed to use hospital number I thought that was sensitive?

File: Your MFT Statisticians/Materials/database setup.Rmd [hunk 4]
```diff
@@ -100,12 +119,16 @@ Frequently data is missing either leave the cell blank or fill it in with n/a. I
 ### What we see
 ![](img/r.PNG){#id .class width=12cm}
 
-In this example when we read in the data all the colours were lost. Feel free to highlight and colour as an aide memoir but do not code in information.
+In this example when we read in the data all the colours were lost. Feel free to highlight and colour as an aide memoir but do not code-in information.
```
Comment (RLHarwood) [inline Your MFT Statisticians/Materials/database setup.Rmd:122]: mémoire is a feminine noun an needs the e also I think it's got a hyphen, consider replacing with memory aid

File: Your MFT Statisticians/Materials/database setup.Rmd (hunk 3 above)
Comment (sgbstats) [reply Your MFT Statisticians/Materials/database setup.Rmd:55]: HN is unique to MFT and we don't have access to any systems where we could use HN so I have always been fine with it.

File: Your MFT Statisticians/Materials/README.md (hunk 1 above)
Comment (sgbstats) [reply Your MFT Statisticians/Materials/README.md:8]: She was fine with this

**Pull request merged**